### PR TITLE
Update FontLoader

### PR DESCRIPTION
**Pull request by jtini**
[SBL-1835](https://sprucelabsai.atlassian.net/browse/SBL-1835)

## Description

- Export the `FontLoader` properly for use in web
- Add `FontLoader` instance to `PageWrapper` for use in skills

## Type

- [* ] Feature
- [ ] Bug
- [ ] Tech debt

### CODE REVIEW

File: packages/spruce-next-helpers/src/skillskit/next/PageWrapper.js [hunk 1]
```diff
@@ -314,12 +342,14 @@ const PageWrapper = Wrapped => {
 			if (this.props.config.DEV_MODE) {
 				return (
 					<Container>
+						<FontLoader fonts={[...fonts]} />
```
Comment (gcpantazis) [inline packages/spruce-next-helpers/src/skillskit/next/PageWrapper.js]: `fonts={fonts}` (unless you have another reason to spread it)

File: packages/spruce-next-helpers/src/skillskit/next/PageWrapper.js [hunk 2]
```diff
@@ -314,12 +342,14 @@ const PageWrapper = Wrapped => {
...
 						<ConnectedWrapped {...this.props} skill={skill} lang={lang} />
 					</Container>
 				)
 			}
 			return (
 				<Container>
+					<FontLoader fonts={[...fonts]} />
```
Comment (gcpantazis) [inline packages/spruce-next-helpers/src/skillskit/next/PageWrapper.js]: ditto

File: packages/spruce-next-helpers/src/skillskit/next/PageWrapper.js (hunk 1 above)
Comment (jtini) [reply packages/spruce-next-helpers/src/skillskit/next/PageWrapper.js]: updated

File: packages/spruce-next-helpers/src/skillskit/next/PageWrapper.js (hunk 2 above)
Comment (jtini) [reply packages/spruce-next-helpers/src/skillskit/next/PageWrapper.js]: updated

**Pull request merged**